### PR TITLE
Property Details API

### PR DESCRIPTION
**Pull request by emilymendelson**
Added property details API at /properties/details/:siteID. This first calls the bcgeocoder Parcel API with SiteID to find associated PIDs. Then it searches for each PID in the ActivePIN database table and returns property details.

https://deloittedigital.atlassian.net/browse/BVPS-120

Request:

- siteID: string
- role: string

Response
- pid
- pin _this is only returned if role is SuperAdmin_
- titleNumber
- landTitleDistrict
- givenName
- lastName_1
- lastName_2
- incorporationNumber
- addressLine_1
- addressLine_2
- city
- province
- otherGeographicDivision
- country
- postalCode

### CODE REVIEW

File: .env.local [hunk 1]
```diff
@@ -10,4 +10,6 @@ TYPEORM_LOGGING = true
 
 # API URL variables
 GEOCODER_API_BASE_URL= 'testurl'
-GEOCODER_API_ADDRESSES_ENDPOINT='endpoint_name.json'
\ No newline at end of file
+GEOCODER_API_ADDRESSES_ENDPOINT='endpoint_name.json'
+BCGEOCODER_API_KEY_PID = 'apikey'
+BCGEOCODER_TEST_API_URL_PID = 'pid testurl'
```
Comment (hannah-macdonald1) [inline .env.local:15]: Is this the test url combined with the endpoint name?

**Review by hannah-macdonald1 — APPROVED** [review]
Just a small comment, otherwise this is good :)

File: .env.local (hunk 1 above)
Comment (emilymendelson) [reply .env.local:15]: Yeah they are combined, but good point I'll split that up

**Review by hannah-macdonald1 — APPROVED** [review]
👍

**Pull request merged**